bool active
int16 cid
ncs_con/obj_msg_2d[] objs
